$OpenBSD: patch-3rdparty_libechonest_CMakeLists_txt,v 1.1 2019/08/11 20:42:44 cwen Exp $

ports-gcc: don't build with -Werror because the code uses dynamic
exception specifications that are deprecated in C++11, thus
generating a warning.

Index: 3rdparty/libechonest/CMakeLists.txt
--- 3rdparty/libechonest/CMakeLists.txt.orig
+++ 3rdparty/libechonest/CMakeLists.txt
@@ -40,10 +40,6 @@ set( LIBECHONEST_H
 
 QT4_WRAP_CPP( ${LIBECHONEST_H} )
 
-if (CMAKE_COMPILER_IS_GNUCXX)
-    add_definitions( -Werror )
-endif (CMAKE_COMPILER_IS_GNUCXX)
-
 SET( OS_SPECIFIC_LINK_LIBRARIES "" )
 IF( APPLE OR MINGW )
   SET( OS_SPECIFIC_LINK_LIBRARIES ${QJSON_LIBRARIES} )
